docs/deprecated.md

```diff
@@ -160,6 +161,14 @@ storage drivers for storing images and containers. This feature was not
 maintained since its inception, and will no longer be supported in upcoming
 releases.
 
+Support for graphdriver plugins is disabled by default in v27.0, but an
+`DOCKERD_DEPRECATED_GRAPHDRIVER_PLUGINS` environment variable is provided
+to re-enable the feature. This environment-variable must be set in the daemon's
```

Comment:
suggestion: "..must be set to a non-empty value in the daemon's environment."

Reply:
nit: also, let's standardize how we write "environment variable" (with or without the `-` between the words?)